snuba/admin/static/data.tsx

```diff
@@ -15,6 +15,7 @@ import MQLQueries from "SnubaAdmin/mql_queries";
 import SnubaExplain from "SnubaAdmin/snuba_explain";
 import Welcome from "SnubaAdmin/welcome";
 import DeleteTool from "SnubaAdmin/delete_tool";
+import ViewCustomJobs from "./manual_jobs";
```

Comment:
Can we follow the same import pattern as other imports? i.e. from `SnubaAdmin` dir?